python/desc/imsim/ImageSimulator.py

```diff
@@ -125,6 +126,10 @@ def _make_gs_interpreters(self, seed, sensor_list, file_id):
                                         self.phot_params,
                                         self.obs_md)
 
+            if self.create_centroid_file is True:
+                self.gs_interpreters[det_name].centroid_base_name = self.outdir \
```

Comment:
It mostly doesn't matter, but it's probably better to use `os.path.join` for concatenating path components.